Program.cs

```diff
@@ -200,6 +247,104 @@ static void Main(string[] args)
             }
         }
 
+        private static int GetMetaDataEpoch(
+            string outputDirectory,
+            string epochType)
+        {
+            try
+            {
+                string previousMetadata = Directory.GetFiles(outputDirectory)
+                    .Where(x => Path.GetExtension(x) == ".json")
+                    .OrderByDescending(x => File.GetLastWriteTime(x))
+                    .First();
+                using (StreamReader reader = new StreamReader(previousMetadata))
+                {
+                    var jsonString = reader.ReadToEnd();
+                    var jsonObject = JObject.Parse(jsonString); 
+                    return (int)jsonObject[epochType];            
+                }
```

Comment:
If you wanted to read all the content of the file, how about using [`File.ReadAllText(string)`](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.readalltext?view=netcore-3.1#System_IO_File_ReadAllText_System_String_) method instead?

```suggestion
                var jsonObject = JObject.Parse(File.ReadAllText(previousMetadata)); 
                return (int)jsonObject[epochType];
```